src/Property/Selector.php

```diff
@@ -37,13 +37,13 @@ class Selector implements Renderable
     private $selector;
 
     /**
-     * @return bool
-     *
      * @internal since V8.8.0
      */
-    public static function isValid(string $selector)
+    public static function isValid(string $selector): bool
     {
-        return \preg_match(static::SELECTOR_VALIDATION_RX, $selector);
+        $numberOfMatches = \preg_match(self::SELECTOR_VALIDATION_RX, $selector);
```

Comment:
This needs to be `static` because `KeyframeSelector` overrides the `SELECTOR_VALIDATION_RX` constant to also allow keyframe animation percentage.

Reply:
Ah, that's why. Fix coming up …